airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/AirbyteMessageMigrationV1.java

```diff
@@ -4,30 +4,69 @@
 
 package io.airbyte.commons.protocol.migrations;
 
+import static io.airbyte.protocol.models.JsonSchemaReferenceTypes.REF_KEY;
+
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ArrayNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.fasterxml.jackson.databind.node.TextNode;
 import io.airbyte.commons.json.Jsons;
 import io.airbyte.commons.version.AirbyteProtocolVersion;
 import io.airbyte.commons.version.Version;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.ConfiguredAirbyteStream;
 import io.airbyte.protocol.models.JsonSchemaReferenceTypes;
+import io.airbyte.protocol.models.v0.AirbyteRecordMessage;
 import io.airbyte.protocol.models.v1.AirbyteMessage;
+import io.airbyte.protocol.models.v1.AirbyteMessage.Type;
```

Comment:
Is there some unexpected changes from the branch rebase? I think this should still be using v-null as v1 instead of the actual v1 namespace.

Reply:
I guess I botched a rebase conflict. looks like this also happened on the upgrades pr (https://github.com/airbytehq/airbyte/pull/19240/files#diff-bca6d63bb992a16edcbc829c7faf7e4321adf0e8b45704ce464453918ea1c961R15); will fix there also